### DISCUSSION TITLE
MODIFIY signup add new field or icon

### DISCUSSION THREAD

**Discussion by 12ahmed12** [hrfee/jfa-go#342] [Q&A]
Hello, can anyone here helps me? i have setup jfa-go for my jellyfin server and everything is working fine 
my concern is i need to update signup page  who some text or icon or add new field in signup  screen 
but dont know how to update the html file! 

btw i read this wiki article and follow guide but couldn't find the right way or path
any help will be appreciated
![image](https://github.com/hrfee/jfa-go/assets/14850196/bc4671c3-b61f-42f1-aed7-c9a6b2842836)

**Comment by hrfee** [comment]
Hi, I believe I had an email conversation with you about this? Im just putting part of my response here to help others:

> The HTML files in the repository --won't-- work by default! I've just added a note about this to the wiki, there is some modification during the build process, so you'll have to base your modifications upon the compiled versions. To save you some time, I've attached them to this email.

and the compiled HTML files (Note these may become outdated, they are from the 26th May):
[html-base.zip](https://github.com/user-attachments/files/16335794/html-base.zip)
